plugins/dynamodb_local/plugin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "dynamodb_local",
+  "version": "0.0.1",
```

Comment:
Should we use a different version here?

(I spotted after this was already merged)

Reply:
Yeah good point, no reason this shouldn't be version 1. It's usable and should be stable for a while